docs/core/porting/cpp-cli-alts.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Migrating C++/CLI projects to ISO C++ and/or .NET
+description: Learn about alternatives to C++/CLI
+author: AaronRobinsonMSFT
+ms.date: 01/30/2023
+---
+
+# Migration from C++/CLI to ISO C++ and/or C\#
+
+The C++/CLI language has been around since 2005 and has enabled users to easily integrate C++ and a .NET language into a single binary. The utility of C++/CLI remains as an innovative way to integrate C++ with the .NET runtime on the Windows platform. There are however limitations to using C++/CLI and as ISO C++ and .NET platforms evolve these limitations are expected to grow.
```

Comment:
> easily

We typically try to avoid this word since it is very subjective. Also, we don't want to oversell something that might not be the best forward-looking choice.